.gitignore

```diff
@@ -2,3 +2,6 @@ site/
 test.py
 __pycache__/
 Thumbs.db
+.DS_STORE
+*sublime-project
```

Comment:
I'll leave that in the `.gitignore` for other people, but dealing with `.sublime-project` really annoyed me. I found [`randy3k/ProjectManager`](https://github.com/randy3k/ProjectManager), and now it's THE plugin I would install if I could only install one plugin (FileManager doesn't count as a plugin of course, it's native :smile:). I really recommend you use it (just rebind `ctrl+alt+p` to his command, and you'll never think about `.sublime-projects` again)

Reply:
I’ll check out that plugin, I feel like the project functions are a bit lacking. So maybe this is a good solution for me.